svgTransform.js

```diff
@@ -2,7 +2,9 @@
 
 module.exports = {
     process() {
-        return 'module.exports = {};'
+        return {
+            code: `module.exports = {};`,
+        }
```

Comment:
have you tested this with svgs ? in my case it was not working

@shayan-deriv can you try rendering the page component in a test file just to make sure that svg's are getting trasnformed correctly ?

Reply:
https://github.com/binary-com/deriv-com/blob/adbd2ae18de53b3b411ccf93bb3ba687c10932b7/__mocks__/svg-mock.js for reference

Reply:
@amir-deriv yes it is working with the new components which are not using styled-component. 

![image](https://github.com/binary-com/deriv-com/assets/100833613/5a9a1bd1-d054-42c8-b8b9-a022b6176617)



but for components which are still using style-component we have a workaround to fix the error.check this file change: 
https://github.com/binary-com/deriv-com/pull/4969/files#diff-fe8049671ea2bf6f8d3af9176022ed50403e26801eb6103eaed1d136a298edc3

but since we are going to remove style-component in the next few months and re-write them using sass , there shouldn't be any worries about this.